saleor/webhook/event_types.py

```diff
@@ -496,6 +496,115 @@ class WebhookEventAsyncType:
         OBSERVABILITY: AppPermission.MANAGE_OBSERVABILITY,
     }
 
+    # ROOT_TYPE mapping is needed by dry-run and manual trigger
+    # of the subscription webhooks events.
+    ROOT_TYPE = {
```

Comment:
Is `ROOT_TYPE` correct name? It is a mapper for supported dry run webhooks, don't know how it is related with `ROOT`